<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>1051</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_g_m">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>281</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>gestion magasin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g_p">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>321</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>gestion parking</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_a_m">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>320</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_del_m">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>320</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_m">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>561</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>ajouter/modifier</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_id_e">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_e">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbv_e">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_spon_e">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID magasin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>nom magasin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>surface</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_e_ajouter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_m_m">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>250</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>350</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>recherche</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_sea_id">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sea_nom">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sea_sur">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SURFACE</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>350</y>
        <width>141</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>tri</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>SURFACE</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_m_pdf">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>370</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>generer pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_m_stat">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>stats</string>
      </property>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>pushButton_a_m</zorder>
     <zorder>pushButton_del_m</zorder>
     <zorder>tableView_m</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>pushButton_m_pdf</zorder>
     <zorder>pushButton_m_stat</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView_p">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>651</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_p_aff">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>300</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>331</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>ajouter/modifier</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_p_id">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_p_nom">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_p_nbrp">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_p_idM">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID PARKING</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>110</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM PARKING</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CAPACITE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID MAGASIN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_p_a">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_p_del">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>250</y>
         <width>81</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_p_supp">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>300</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>320</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>recherche</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_p_sea_id">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit__p_sea_nom">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_p_sea_cap">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_P</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CAPACITE</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>320</y>
        <width>161</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_p_triID">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_p_trinom">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_p_tricp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>CAPACITE</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_p_genp">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>360</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>tableView_p</zorder>
     <zorder>pushButton_p_supp</zorder>
     <zorder>groupBox_5</zorder>
     <zorder>groupBox_6</zorder>
     <zorder>pushButton_p_genp</zorder>
     <zorder>pushButton_p_aff</zorder>
     <zorder>statP</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QCustomPlot" name="customplot" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>841</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_s_m">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>return</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>450</y>
        <width>167</width>
        <height>29</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>main menu</string>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
